cs-nutanix-prism/src/main/java/io/cloudslang/content/nutanix/prism/actions/virtualmachines/AddNic.java

```diff
@@ -0,0 +1,155 @@
+import static io.cloudslang.content.nutanix.prism.utils.HttpUtils.getOperationResults;
+import static io.cloudslang.content.nutanix.prism.utils.Inputs.CommonInputs.PASSWORD;
+import static io.cloudslang.content.nutanix.prism.utils.Inputs.CommonInputs.PROXY_HOST;
+import static io.cloudslang.content.nutanix.prism.utils.Inputs.CommonInputs.PROXY_PASSWORD;
+import static io.cloudslang.content.nutanix.prism.utils.Inputs.CommonInputs.PROXY_PORT;
+import static io.cloudslang.content.nutanix.prism.utils.Inputs.CommonInputs.PROXY_USERNAME;
+import static io.cloudslang.content.nutanix.prism.utils.Inputs.CommonInputs.USERNAME;
+import static io.cloudslang.content.nutanix.prism.utils.Inputs.CommonInputs.*;
+import static io.cloudslang.content.nutanix.prism.utils.Inputs.GetVMDetailsInputs.VM_UUID;
+import static io.cloudslang.content.nutanix.prism.utils.InputsValidation.verifyCommonInputs;
+import static io.cloudslang.content.nutanix.prism.utils.Outputs.CommonOutputs.TASK_UUID;
+import static io.cloudslang.content.utils.OutputUtilities.getFailureResultsMap;
+import static org.apache.commons.lang3.StringUtils.EMPTY;
+import static org.apache.commons.lang3.StringUtils.defaultIfEmpty;
+
+public class AddNic {
+    @Action(name = ADD_NIC_OPERATION_NAME,
+            description = ADD_NIC_SUCCESS_DESC,
+            outputs = {
+                    @Output(value = RETURN_RESULT, description = RETURN_RESULT_DESC),
+                    @Output(value = EXCEPTION, description = EXCEPTION_DESC),
+                    @Output(value = STATUS_CODE, description = STATUS_CODE_DESC),
+                    @Output(value = TASK_UUID, description = TASK_UUID_DESC)
+            },
+            responses = {
+                    @Response(text = SUCCESS, field = RETURN_CODE, value = ReturnCodes.SUCCESS, matchType = COMPARE_EQUAL,
+                            responseType = RESOLVED, description = SUCCESS_DESC),
+                    @Response(text = FAILURE, field = RETURN_CODE, value = ReturnCodes.FAILURE, matchType = COMPARE_EQUAL,
+                            responseType = ERROR, description = FAILURE_DESC)
+            })
+    public Map<String, String> execute(@Param(value = HOSTNAME, required = true, description = HOSTNAME_DESC) String hostname,
+                                       @Param(value = PORT, description = PORT_DESC) String port,
+                                       @Param(value = USERNAME, required = true, description = USERNAME_DESC) String username,
+                                       @Param(value = PASSWORD, encrypted = true, required = true, description = PASSWORD_DESC) String password,
+                                       @Param(value = VM_UUID, required = true, description = VM_UUID_DESC) String vmUUID,
+                                       @Param(value = Inputs.CreateNicInput.NETWORK_UUID, required = true, description = Descriptions.CreateNic.NETWORK_UUID_DESC) String networkUUID,
```

Comment:
Be consistent with Class and Action name
Action Name: AddNic
Input Class name: CreateNicInput
either change class names/methods to AddNicInput or change Action Name.

And input variable name also should be Consistent
vlanid --> vlanId